Test.java

```diff
@@ -0,0 +1,39 @@
+package com.baluybs.shop;
+import java.util.Scanner;
+
+public class Test {
+
+	public static void main(String[] args) {
+		Scanner sc=new Scanner(System.in);
+		while(true) {
+			System.out.println("Press 1 for List of Products");
+			System.out.println("Press 2 to add Products");
+			System.out.println("Press 3 to remove Products");
+			System.out.println("Press 4 to view Cart");
+			System.out.println("enter option");
+			int opt=sc.nextInt();
+			select(opt);
+		}
+	}
+	public static void select(int opt) {
```

Comment:
Created a separate function `Good`
but forgot to use a proper naming convention.